viper-runner/scripts/after_plugin.vpr

```diff
@@ -0,0 +1,11 @@
+field left: Int
+field right: Int
+  
+// After plugin
```

Comment:
Test `.vpr` file for benchmarking what life would be like to exist _with_ the inline predicate plug in.